gunrock/applications/sssp.hxx

```diff
@@ -16,32 +16,28 @@
 namespace gunrock {
 namespace sssp {
 
-template <typename vertex_t>
+template <typename graph_t>
```

Comment:
Same as above -- I like these taking `graph_t` instead of a list of types that are needed.  Thoughts?